testing/Dockerfile

```diff
@@ -53,8 +53,10 @@ RUN chmod a+x /usr/local/bin/bootstrap.sh
 
 # We could pre-load requirements to make running the tests
 # faster, but this requires rebuilding every time we have a new dep.
-COPY staging/requirements-cpu.txt /tmp/requirements-cpu.txt
+COPY staging/requirements.txt /tmp/requirements.txt
```

Comment:
why did this go into /tmp?

Reply:
Oh, I dunno. Made the workdir /testing and put it there.

Reply:
seems fine.